src/storybook-docs/installation.stories.mdx

```diff
@@ -0,0 +1,65 @@
+import { Meta, Story, Canvas } from '@storybook/addon-docs/blocks';
+
+<Meta title="Instalasi" />
+
+# Selamat Datang di Jabar Digital Service Sistem Desain
+
+**Jabar Digital Service Sistem Desain** adalah _open source_ JDS yang digunakan dalam mendesain dan mengembangkan aplikasi web maupun mobile dalam lingkungan pemerintahan Jawa Barat. Sistem desain ini terdiri dari komponen siap pakai dan kumpulan kode pendukungnya serta pedoman penggunaan untuk masing-masing komponen tersebut.
+
+## Catatan
+Sistem desain ini masih diperuntukan digunakan pada *framework* VueJS selain itu kami juga berharap agar bisa mendukung penggunaan di *framework* lain dengan membuat *library* ini menjadi *web component* yang sedang kami kembangkan sampat saat ini.
+
+## Instalasi
+Untuk melakukan instalasi sistem desain ini anda dapat melakukannya dengan menggunakan aplikasi *package manager* NPM, lakukan perintah dibawah ini dengan aplikasi **CLI** (**C**ommand **L**ine **I**nterface) pada root direktori project anda.
+
+```bash 
+npm i @jabardigitalservice/jds-design-system
+```
+setelah anda berhasil melakukan instalasi, buka file main.js yang berarti *entrypoint* pada project anda, lalu tambahkan kode dibawah ini.
+```javascript 
+import { DesignSystem } from '@jabardigitalservice/jds-design-system'
+Vue.use(DesignSystem)
+```
+### Desain Token
+Apabila dan ingin menggunakan desain token kami, anda dapat menambahkan kode di bawah pada *entrypoint* main.js
+```javascript
+import '@jabardigitalservice/jds-design-system/css/jds-design-system.css'
+```
+### Icons
+Apabila anda ingin menggunakan icons sistem kami, anda dapat menambahkan kode di bawah pada *entrypoint* main.js
```

Comment:
To be discussed:
some words in this page using english or term words, so is it
icons or _icons_
root or _root_
library or _library_ or librari
diperuntukan or diperuntukkan
and in my opinion, sistem desain is kind of weird, so is it sistem desain or desain sistem

thanks before

Reply:
Okay thanks @yoslie,
first of all i will correct your english about "thanks before" mistake to use it, because that never use in native. you can use "Thanks In Advance"

the documentations we provide guide with bahasa indonesia, so all of english word we cannot translate to bahasa will be italic format, but if we can translate word to bahasa and that the word usual in bahasa we write that in bahasa.

- icons
- root
- project
- web
- mobile

that my mistake, it should be italic formated, i will fix it ASAP, and about "sistem desain" i have copied from documentations installation @adrianpdm create before, it should be discussed with @adrianpdm too.